io/disk/ssd/nvmetest.py

```diff
@@ -44,6 +45,7 @@ def setUp(self):
         """
         self.device = self.params.get('device', default='nvme0')
         self.device = "/dev/%s" % self.device
+        self.ns_count = self.params.get('namespace_count', default=1)
```

Comment:
can  this be moved in test itself and having variable scop local

Reply:
@PraveenPenguin we will be needing this in other test too in future, as complete test structure will be going to change going forward.
So I feel having in setup will help better.

Reply:
no, still I feel whenever requirement we should ACT not on a futuristic patch  :) , current one will be based on today's requirement